src/main/java/core/org/akaza/openclinica/web/filter/ApiSecurityFilter.java

```diff
@@ -229,97 +226,35 @@ public OCUserDTO getUserDetails (HttpServletRequest request) {
         return map;
     }
 
-    //TODO Put this somewhere else?
-    private HashMap<String, String>  createRandomizeUserAccount() {
```

Comment:
How is this going to impact existing customers that have randomize and dicom users already created? Do we need to write an upgrade script to migrate those accounts?